frontend/components/text-classifier/labeling-rules/RuleDefinition.vue

```diff
@@ -94,6 +100,27 @@ export default {
     async updateRule() {
       await this.$fetch();
     },
+    async resetFilters() {
+      let query = {};
+      query = Object.keys(this.dataset.query)
+        .filter((k) => k !== "text")
```

Comment:
Maybe we can reset all filters, including the text query, giving more feeling of initial state